lib/appfigures/client.rb

```diff
@@ -43,29 +43,116 @@ def product_keys
     def usage
       begin
         _, resp = do_get(AppFigures::API::USAGE)
-        resp
+        resp[:body]
       rescue Exception => e
         puts e.message
         puts e.backtrace
       end
     end
 
-    def ranks(args = {})
+    # http://docs.appfigures.com/products
+    # does not support search
```

Comment:
Comment is to describe what is the functionality and how to use.
If we want to note this one, it's better to have a `note` section.
For example, it can be like
```
#
# Get products info from AppFigures
#
# ...
# ...
#
# See http://docs.appfigures.com/products for detail
# Note that AppFigures#products does not support search
#
```